### PR TITLE
LPS-120407 Show home icon after hiding the landing page

### PR DESCRIPTION
**Pull request by LuismiBarcos**
I've renamed the attributes named `category` to `section` instead of `topic` to be consistent with the code of Questions.
Rename `section` to `topic` along all the app is very risky, more if we are near to the release because is very easy to include a critical bug.
So, we think is better to include this rename in our task about technical debt ([LPS-109174](https://issues.liferay.com/browse/LPS-109174)).

### CODE REVIEW

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/BreadcrumbDropdown.es.js [hunk 1]
```diff
@@ -23,43 +23,48 @@ export default ({createLink = true, section, ...props}) => {
...
 
 	return (
-		<ClayDropDown
-			active={active}
-			onActiveChange={setActive}
-			trigger={
-				<ClayButton
-					className="c-p-0 questions-breadcrumb-unstyled text-truncate"
-					displayType="unstyled"
+		<>
```
[the remark this answers is not in the document]
Comment (LuismiBarcos) [reply modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/BreadcrumbDropdown.es.js]: Yes, you are right! I've changed the name of the file `BreadcrumbDropdown` to `BreadcrumbNode` and changed the code a little bit to convert it in a component that represents a node of the breadcrumb.

**Review by javierdearcos — APPROVED** [review]
👏 👏 👏

**Comment by LuismiBarcos** [comment]
ci:forward

**Comment by javierdearcos** [comment]
ci:forward

**Comment by LuismiBarcos** [comment]
ci:forward

**Pull request closed without merging**